packages/ui/src/ui/utils/system/masters.ts

```diff
@@ -0,0 +1,13 @@
+import {makeRegexpFromSettings} from '../../../shared/utils';
+import {uiSettings} from '../../config/ui-settings';
+
+export const calcMasterShortName = (address: string) => {
```

Comment:
The function looks like a copy-paste from `Host.tsx`, it is better to have only one version.